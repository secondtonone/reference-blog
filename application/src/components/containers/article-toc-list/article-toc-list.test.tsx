/**
 * @jest-environment jsdom
 */
import toJson from 'enzyme-to-json';
import { mount } from 'enzyme';
import ArticleTocList from '.';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      asPath: '/blog/test-article'
    };
  },
}));

describe('Match snapshots ArticleTocList', () => {
  it('default', () => {
    let tree = null;
    tree = mount(
      <ArticleTocList>
        <ul>
          <li><a href="#header1">Header 1</a></li>
          <li><a href="#header2">Header 2</a></li>
        </ul>
      </ArticleTocList>
    );

    expect(toJson(tree)).toMatchSnapshot();
  });
});
