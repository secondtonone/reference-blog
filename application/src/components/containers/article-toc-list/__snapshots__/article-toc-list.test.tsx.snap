// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshots ArticleTocList default 1`] = `
<ArticleTocList>
  <article-toc-list__Toc
    onClick={[Function]}
  >
    <div
      className="article-toc-list__Toc-sc-1q5d9b4-0 cfLcBw"
      onClick={[Function]}
    >
      <ul>
        <li>
          <a
            href="#header1"
          >
            Header 1
          </a>
        </li>
        <li>
          <a
            href="#header2"
          >
            Header 2
          </a>
        </li>
      </ul>
    </div>
  </article-toc-list__Toc>
</ArticleTocList>
`;
